app/admin/user.rb

```diff
@@ -21,6 +21,16 @@ def update
       end
       super
     end
+
+    def destroy
+      # Before destroy, Checking associated
+      user = User.find(params[:id])
```

Comment:
isnt this default behaviour? I always get errors without exending this function. Could you find out what the differance is between this function and just the default behaviour